src/main/groovy/io/openliberty/tools/gradle/tasks/AbstractFeatureTask.groovy

```diff
@@ -291,5 +293,17 @@ public class AbstractFeatureTask extends AbstractServerTask {
         return util
     }
 
+    protected void setContainerEngine(AbstractContainerSupportUtil util) throws PluginExecutionException {
+        String LIBERTY_DEV_PODMAN = "liberty.dev.podman";
+        Map<String, Object> projectProperties = project.getProperties();
+        if (!projectProperties.isEmpty()) {
+            Object isPodman = projectProperties.get(LIBERTY_DEV_PODMAN);
+            isPodman = projectProperties.get(LIBERTY_DEV_PODMAN);
+            if (isPodman != null) {
+                util.setIsDocker(!(Boolean.parseBoolean(isPodman.toString())));
+                logger.debug("liberty.dev.podman was set to: " + (Boolean.parseBoolean(isPodman.toString())));
```

Comment:
Same comments as for LMP (why get the property twice, why not use containsKey, and check the string for a valid true/false and give warning before calling `setIsDocker`)